code/__DEFINES/food.dm

```diff
@@ -15,9 +15,10 @@
 #define CLOTH (1<<14)
 /*#define NUTS (1<<15)
 #define SEAFOOD (1<<16)
-#define ORANGES (1<<17)
+#define CITRUS (1<<17)
 #define BUGS (1<<18)*/
 #define GORE (1<<19)
+#define POLYMER (1<<20)
```

Comment:
Is this even used anywhere?

Reply:
Removed, tempted to add a new "food taste stuff, backfire in testing"